tests/integration/recordings/responses/0002a233aedd.json

```diff
@@ -49,7 +49,7 @@
               "logprobs": null
             }
           ],
-          "created": 1759368459,
+          "created": 0,
```

Comment:
nit: why is created: 0 needed?

Reply:
@raghotham it is a source of noise if you re-record you will get a new timestamp. it is not salient information.